code/_onclick/hud/screen_objects.dm

```diff
@@ -486,7 +488,7 @@
 				var/mob/living/silicon/ai/AI = usr
 				AI.view_images()
 		else
-			return 0
+			return attempt_vr(src,"Click_vr",list(location,control,params)) //VOREStation Add - Additional things.
```

Comment:
Is this supposed to be here?

Reply:
Nnnnnope. I'm just going to fix this and that thing I did in that accidental .dme thing and forcepush over this branch. Be riiiight back.